MailCore/Models/Message.swift

```diff
@@ -111,6 +114,7 @@ public struct MessagesUids {
     public var deletedUids = [String]()
     public var updated = [MessageFlags]()
     public let cursor: String
+    public var folderUnreadCount: Int? = nil
```

Comment:
redundant_optional_initialization
```suggestion
    public var folderUnreadCount: Int?
```